python/build_cache.py

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env python
 
 import os
+from pathlib import Path
```

Comment:
Just so we are on the same page, the caching doesn't work, but I left it in here in case we wanted to use it...